data/data/install.openshift.io_installconfigs.yaml

```diff
@@ -1706,6 +1710,7 @@ spec:
                 required:
                 - apiVIP
                 - ingressVIP
+                - interfaceBindingMethod
```

Comment:
I'm not sure we want it to be required, such change will break existing scripts and tests

Reply:
Right.
The intention was to make it optional, with default setting for BridgeInterface, for backward compatibility.
I've updated the code.